### PR TITLE
insun week6 homework

### PR DESCRIPTION
**Pull request by hwanginsun**
6주차 과제입니다.

이번 과제는 tensorflow를 활용하여 DNN을 구성해서 emnist dataset을 학습하는 내용입니다

DNN을 설계할 때 Deep & Wide, learning_rate, epoch 횟수, 초기화 방법 등의 parameter에 따라 많은 영향이 있다는 것을 알았습니다.

무조건 깊고 넓은게 좋은게 아니라 각 dataset에 맞도록 구성해야 하는 것을 배웠습니다.

그리고 image처리에 앞서서 image와 label을 제대로 구분했는지 확인 작업이 중요하다는 것을 알았습니다.

문제에서 요구하는대로 test set에 대해 정확도 90%를 넘기지는 못했지만, 앞으로 배울 내용을 통해 개선할 수 있을 것으로 기대하고 있습니다.

< train set 훈련 결과 >
image

< accuracy and loss >
image

< test set >
image
  image

### CODE REVIEW

**Comment by anthony0727** [comment]
잘 하셨습니다! :)
